src/main/java/com/benmorant/game/holycraap/model/entity/People.java

```diff
@@ -33,7 +37,54 @@
   private int currentMp;
   private int mpMax;
 
-  @OneToMany
-  @JoinColumn(name = "people_id")
-  private List<Item> inventory;
+  @OneToMany(mappedBy = "people")
+  @JsonManagedReference
+  private List<Item> inventory = new ArrayList<>();
```

Comment:
This attribute should be `final`, this will make sure `inventory` is always the same object/reference, which makes it easier to pass around.

Reply:
done